### PR TITLE
new esri leaflet browserify sample

### PR DESCRIPTION
**Pull request by jgravois**
a quick review/merge would be greatly appreciated.  i'm working on publishing a blog post that will need to reference the live sample.  

i'm not asking anyone to push the merge button before they've been able to run the sample locally and understand what its demonstrating.  in fact, **critical** feedback about whats unclear in the current sample would also be warmly received.

### CODE REVIEW

File: web-leaflet/browserify/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+## Prerequisites
+
+install [Node.js](https://nodejs.org/en/)/npm.
+
+## Development Steps
+
+1. fork/clone this github repository
+2. navigate into this folder using a command prompt and run `npm install`
+> this will lay down the project [dependencies](https://github.com/Esri/developer-support/blob/master/web-leaflet/browserify/package.json#L8-L18)
```
Comment (jgravois) [inline web-leaflet/browserify/README.md]: links like this will resolve after the sample is merged.  for now:
s/Esri/jgravois
s/master/leaflet-browserify

**Comment by NoashX** [comment]
:ok_hand: totally works for me, and I quite like the browserify abilities...

My only concern is a warning indicating that an optional dependency failed:

npmwarn

is this expected?

**Comment by jgravois** [comment]
the warning is a little less cryptic on a mac :smile:

screenshot 2015-10-14 11 09 11

this is what i consider the best of two alternatives.  people do often install `http-server` globally so that they can spin up the server from any folder, so i could have definitely added an extra step to the developer instructions asking people to run `npm install http-server -g`.

that being said, personally, i'm not a huge fan of global modules (see [here](http://firstdoit.com/npm-scripts/)) and like the idea that the sample is entirely self contained.  because of this, it seemed like the lesser of two evils to suffer the warning.

**Comment by NoashX** [comment]
Ahh, the old macBook pro ftw routine :computer:

Makes sense to me, global seems unnecessary. Merging with confidence.

**Pull request merged**